src/PrestaShopBundle/Controller/Admin/Sell/Catalog/CategoryController.php

```diff
@@ -92,7 +92,7 @@ public function indexAction(Request $request, CategoryFilters $filters)
         $categoryViewDataProvider = $this->get('prestashop.adapter.category.category_view_data_provider');
         $categoryViewData = $categoryViewDataProvider->getViewData($currentCategoryId);
 
-        $deleteCategoriesForm = $this->createForm(DeleteCategoriesType::class);
+        $deleteCategoriesForm = $this->createForm(DeleteCategoriesType::class, ['categories_to_delete_parent' => (int) $request->query->get('categoryId')], []);
```

Comment:
Mmm, I believe `$request->query` is the ParameterBag for POST request, for GET request it's supposed to be stored in `$request->request`
Or alternatively you can do `$request->get()` directly which combines both (along with route attributes)

Reply:
My bad it's the opposite ^^ `$request->query` => GET, `$request->request` => POST

Reply:
Hi @jolelievre 
We don't really need all of that ... Wdyf ?
See : https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Controller/Admin/Sell/Catalog/CategoryController.php#L91